velox/row/UnsafeRowDeserializers.h

```diff
@@ -528,6 +529,23 @@ struct UnsafeRowPrimitiveBatchDeserializer {
   static StringView deserializeStringView(std::string_view data) {
     return StringView(data.data(), data.size());
   }
+
+  static int128_t deserializeLongDecimal(std::string_view data) {
+    const uint8_t* bytesValue = reinterpret_cast<const uint8_t*>(data.data());
+    int length = data.size();
+    uint8_t bytesValue2[32]{};
```

Comment:
naming: bytesValue2 -> bytes or something else that does't have '2' at the end

What does {} do here?

Reply:
https://stackoverflow.com/questions/38892455/initializing-an-array-of-zeroes
initialize the array to 0